configs/_base_/datasets/dota_coco.py

```diff
@@ -0,0 +1,96 @@
+# dataset settings
+dataset_type = 'mmdet.CocoDataset'
+data_root = 'data/split_ms_dota/'
+file_client_args = dict(backend='disk')
+
+train_pipeline = [
+    dict(type='mmdet.LoadImageFromFile', file_client_args=file_client_args),
+    dict(type='mmdet.LoadAnnotations', with_bbox=True, with_mask=True),
```

Comment:
We can set poly2mask=False to load polygon mask. There is no need to convert the annotation to bitmap mask and then convert it to rbox.

Reply:
Very reasonable!